__tests__/books.test.js

```diff
@@ -0,0 +1,69 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const Book = require('../lib/models/Book');
+
+describe('01-backend-hand-of-resources books route', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  afterAll(() => {
+    pool.end();
+  });
+
+  test('create a new book', async () => {
+    const expected = {
+      title: 'book',
+      author: 'author',
+      bookNumber: 1,
+      seriesLength: 1,
+    };
+    const res = await request(app).post('/api/v1/books').send(expected);
+
+    expect(res.body).toEqual({ id: expect.any(Number), ...expected });
+  });
+
+  test('fetch all books', async () => {
+    const expected = await Book.findAll();
```

Comment:
A stronger test would be to insert two books, and then your assertion would be `expect(res.body).toEqual([book1, book2])`